plugins/scaffolder-backend/src/lib/files/serializeDirectoryContents.ts

```diff
@@ -62,19 +61,11 @@ export async function serializeDirectoryContents(
 
   const limiter = limiterFactory(10);
 
-  const valid = await asyncFilter(paths, async ({ dirent, path }) => {
+  const valid = await asyncFilter(paths, async ({ dirent }) => {
     if (dirent.isDirectory()) return false;
     if (!dirent.isSymbolicLink()) return true;
 
-    const safePath = resolveSafeChildPath(sourcePath, path);
```

Comment:
So I think that this needs to stay. As we don't want to allow creating symlinks to things like `/etc/passwd` or `~/.aws/credentials` for security reasons.